.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npm run lint
```

Comment:
we should be checking only the staged files

- npm lint-staged
- you would need to install lint-staged as well

Also lets take a note that after adding prettier we should include it here as well